test/lib/ufe/ufeTestUtils/mayaUtils.py

```diff
@@ -127,3 +130,17 @@ def openTopLayerScene():
     filePath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "test-samples", "ballset", "StandaloneScene", "top_layer.ma" )
     cmds.file(filePath, force=True, open=True)
     
+def previewReleaseVersion():
```

Comment:
Utility function to return the Maya preview release version.  The parenting test must not be run for a Maya version before preview release 115 (unless you use the special override environment variable).